pr_agent/cli.py

```diff
@@ -23,6 +25,10 @@ def run():
         print(f"PR description: {args.pr_url}")
         reviewer = PRDescription(args.pr_url)
         asyncio.run(reviewer.describe())
+    elif args.pr_code_suggestions:
+        print(f"PR code suggestions: {args.pr_url}")
+        reviewer = PRCodeSuggestions(args.pr_url)
+        asyncio.run(reviewer.suggest())
     else:
         print(f"Reviewing PR: {args.pr_url}")
         reviewer = PRReviewer(args.pr_url, cli_mode=True)
```

Comment:
**Suggestion:** Consider using a dictionary to map command line arguments to functions. This will make it easier to add new commands in the future.
```suggestion
    commands = {
        'question': PRQuestions,
        'pr_description': PRDescription,
        'pr_code_suggestions': PRCodeSuggestions,
        'default': PRReviewer
    }
    command = commands.get(args.command, commands['default'])
    reviewer = command(args.pr_url)
    asyncio.run(reviewer.run())
```